src/sdk/service/network-validation-service/network-validation-service.ts

```diff
@@ -0,0 +1,468 @@
+import { IPv4AddressRange, IPv4NetworkDescriptor } from './network-validation-interface';
+
+/**
+ * Network Validation Service
+ */
+export abstract class NetworkValidationService {
+  /**
+   * Gets the integer representation of an IP address.
+   * @param {string} ip: the ip address.
```

Comment:
there should not be a colon after the param name. comment applies throughout the PR

Reply:
Would a hyphen be better or should I literally put nothing? @cfsnyder